src/components/address-book/ImportDialog/validation.ts

```diff
@@ -2,11 +2,7 @@ import type { ParseResult } from 'papaparse'
 
 import { validateAddress, validateChainId } from '@/utils/validation'
 
-export const abCsvReaderValidator = ({ type, size }: File): string[] | undefined => {
-  if (type !== 'text/csv') {
-    return ['Address book must be a CSV file']
-  }
-
+export const abCsvReaderValidator = ({ size }: File): string[] | undefined => {
```

Comment:
This is not a fix, right? Either we investigate deeper or I'd say let's not fix this.

Reply:
Let's see what Franco says. If it fixes it, I'd say it's better not to do the validation that gives false negatives on some OSes.

Reply:
Ok, if we are having false negatives in some OSes. Waiting for Franco on this.